unix_commands.sh

```diff
@@ -0,0 +1,14 @@
+echo Shaked >> a-1
+touch 1-b
+echo Shaked >> 1-b
+mkdir new
+cp 1-a new/
+mv 1-a 1-a-new
+gedit 1-a-new
+Shaked, cats
+echo Shaked, cats >> new/newFile
+cd new
+ls
+cd .. 
+cd -
```

Comment:
instead of using cd -, you can use cd with the tilde (~) symbol to switch back to the home directory.